tools/html_transformer/html_transformer.js

```diff
@@ -0,0 +1,95 @@
+var Promise = require('es6-promise').Promise;
+var jsdom = require('jsdom');
+
+exports.window = null;
+
+exports.load = function(filename) {
+  return new Promise(function(resolve, reject) {
+    jsdom.env(filename, function(errors, window) {
+      if (errors && errors.length) {
+        return reject(errors[0]);
+      }
+
+      exports.window = window;
+      return resolve();
```

Comment:
I tend to prefer less indentation which is why I usually go for the `exports` style thing rather than `module.exports` when I'm exporting multiple things